### PR TITLE
[FL-2780] NFC Notifications fix

### PR DESCRIPTION
**Pull request by Astrrra**
# What's new

- Notifications changed to match the Miro board

# Verification 

- Read and emulate every type of card, confirm that the behaviour matches the board.

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/nfc/nfc.c [hunk 1]
```diff
@@ -224,6 +232,21 @@ void nfc_show_loading_popup(void* context, bool show) {
 }
 
 int32_t nfc_app(void* p) {
+    if(!furi_hal_nfc_is_init()) {
```
Comment (gornekich) [inline applications/nfc/nfc.c]: One PR - one feature. Please, checkout from dev

**Review by gornekich — CHANGES_REQUESTED** [review]
Everything is good, just delete code that is not for this task

File: applications/nfc/nfc.c (hunk 1 above)
Comment (Astrrra) [reply applications/nfc/nfc.c]: Oh, I think I accidentally checked out the wrong branch, yep

Comment (Astrrra) [reply applications/nfc/nfc.c]: Fixed!

**Review by gornekich — DISMISSED** [review]

**Review by skotopes — APPROVED** [review]

**Pull request merged**